java/core/src/test/java/co/worklytics/psoxy/rules/msft/TeamsTests.java

```diff
@@ -0,0 +1,333 @@
+package co.worklytics.psoxy.rules.msft;
+
+import co.worklytics.psoxy.rules.JavaRulesTestBaseCase;
+import co.worklytics.psoxy.rules.RESTRules;
+import co.worklytics.psoxy.rules.Rules2;
+import jdk.jfr.Description;
+import lombok.Getter;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class TeamsTests extends JavaRulesTestBaseCase {
+
+    @Getter
+    final Rules2 rulesUnderTest = PrebuiltSanitizerRules.MS_TEAMS;
+
+    @Getter
+    final RulesTestSpec rulesTestSpec = RulesTestSpec.builder()
+        .sourceFamily("microsoft-365")
+        .defaultScopeId("azure-ad")
+        .sourceKind("ms-teams")
```

Comment:
This should match https://github.com/Worklytics/evalengin/blob/working/worklytics-common/src/main/java/com/engetc/collaboration/msft/teams/TeamsIntegration.java#L20

Reply:
Done